<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Issues</class>
 <widget class="QDialog" name="Issues">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Report Issue</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Users/brigh/OneDrive/Pictures/My_Covid_Record_Logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="title">
      <string>Do you have any issues?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>9</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>User Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="userName"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>9</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Email Address</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="emailAddress"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>9</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Issues</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>65</red>
              <green>140</green>
              <blue>159</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>88</red>
              <green>179</green>
              <blue>211</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>240</red>
              <green>240</green>
              <blue>240</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>65</red>
              <green>140</green>
              <blue>159</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>120</red>
              <green>120</green>
              <blue>120</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>11</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>&lt;a href=&quot;mailto:issuesupport@covid.co.nz?subject=Report issue&quot;&gt;Send email to issuesupport@covid.co.nz&lt;/a&gt;</string>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
